src/main/frontend/components/block.cljs

```diff
@@ -762,15 +762,12 @@
 
         (= name "youtube")
         (let [url (first arguments)]
+          ;; javascript regex "^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w\-]+\?v=|embed\/|v\/)?)(?<id>[\w\-]+)(\S+)?$"
```

Comment:
Let's remove this line if it's not used :)

Reply:
done